### DISCUSSION TITLE
top view 2d weather sandbox

### DISCUSSION THREAD

**Discussion by maxrobux** [niels747/2D-Weather-Sandbox#73] [Ideas]
if it possible can make the top view of weather sandbox?

**Comment by odvigajlo** [comment]
i'm not sure it's possible to make "top view" simulation without having actual 3d simulation. Just because thunderstorms are really based of their height which cannot be simulated in 2d "top view"

**Comment by niels747** [comment]
From the top it would just be a single line...
I have thought about some sort of a minimap.
